openvdb/cmd/openvdb_print.cc

```diff
@@ -141,7 +245,18 @@ printLongListing(const StringVec& filenames)
                 grid->print(std::cout, /*verboseLevel=*/11);
                 firstGrid = false;
             }
+#if OPENVDB_ABI_VERSION_NUMBER >= 6
+            // Check for grid type, inspect points if this is a point grid
+            const openvdb::GridBase::ConstPtr grid = *it;
+            Name gridType = grid->type();
+            if (gridType.find("ptdata") != std::string::npos)
```

Comment:
Don't bother with this nametype check, you can call `GridBase::grid<T>` with the grid and check the resulting ptr